SMLHelper/Patchers/LanguagePatcher.cs

```diff
@@ -12,8 +12,6 @@ internal class LanguagePatcher
         private static readonly string LanguageDir = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Language");
         private static readonly string LanguageOrigDir = Path.Combine(LanguageDir, "/Originals");
         private static readonly string LanguageOverDir = Path.Combine(LanguageDir, "/Overrides");
```

Comment:
The language patcher currently doesn't work correctly and returns 0 files. Will have to fix that before merging.

Reply:
We can use our last attempt instead for now (which has been proven to work) and change it later.  
https://github.com/SMLHelper/SMLHelper/pull/92

Reply:
After 22b8013, works perfectly. The only current issue(?) is that the curly braces are added as part of the text for old files.

Reply:
But I don't think that's an "issue". People can update their files.